src/app/run/[courseId]/[ghostRunningId].tsx

```
@@ -288,14 +303,23 @@ export default function Run() {
                 value: getFormattedPace(context.stats.avgPaceSecPerKm ?? 0),
             },
             {
-                description: "케이던스",
-                value: Math.round(context.stats.avgCadenceSpm ?? 0),
-                unit: "spm",
+                description: "케이던스(spm)",
+                value:
+                    context.stats.avgCadenceSpm ?? 0 > 0
+                        ? Math.round(context.stats.avgCadenceSpm ?? 0)
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)